src/Common/ItechArt.Common/Logging/LoggerExtension.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+namespace ItechArt.Common.Logging;
+
+public static class LoggerExtension
```

Comment:
1. move to extensions folder
2. move service collection extensions to Logging folder root